rule-engine/rule-engine-components/src/main/java/org/thingsboard/rule/engine/deduplication/TbMsgDeduplicationNodeConfiguration.java

```diff
@@ -21,6 +21,8 @@
 @Data
 public class TbMsgDeduplicationNodeConfiguration implements NodeConfiguration<TbMsgDeduplicationNodeConfiguration> {
 
+    private static final String DEFAULT_QUEUE_NAME = "Main";
```

Comment:
Extract the "Main" to DataConstants and use here and in all other places where we use "Main".